panels/dev-info/ha-panel-dev-info.html

```diff
@@ -41,7 +46,6 @@
 
       .error-log-intro {
         margin-top: 16px;
-        border-top: 1px solid var(--light-primary-color);
```

Comment:
If you're removing the border, remove the padding top and set margin to 24px.

Reply:
Changed, I hope I understood you correctly.